service/api_logic/filter_manager/common_filters.py

```diff
@@ -0,0 +1,22 @@
+from database.models import League
+from sqlalchemy.orm import Query
+from service.api_logic.scripts import get_sport_by_name
+
+class CommonFilters():
```

Comment:
I don't think we need a common filters class. At least not in its current condition.

Or at least rename it from "Common..." to smth else